controllers/storageconsumer/storageconsumer_upgrade_controller.go

```diff
@@ -219,6 +220,13 @@ func (r *StorageConsumerUpgradeReconciler) reconcileStorageRequest(
 	rbdStorageRequest.SetGroupVersionKind(ocsv1alpha1.GroupVersion.WithKind("StorageRequest"))
 	rbdStorageRequest.Name = rbdStorageRequestName
 	rbdStorageRequest.Namespace = storageConsumer.Namespace
+	if err := r.Client.Get(ctx, client.ObjectKeyFromObject(rbdStorageRequest), rbdStorageRequest); client.IgnoreNotFound(err) != nil {
+		return err
+	}
+	rbdStorageRequest.Finalizers = []string{}
+	if err := r.Client.Update(ctx, rbdStorageRequest); client.IgnoreNotFound(err) != nil {
```

Comment:
just reminding again 😅, this is partial meta and update will fail on this, you need a patch 😄.